cypress/integration/AdvancedSearchPage/LocationSection/LocationLimitedToVA.feature

```diff
@@ -23,7 +23,7 @@ Scenario: User has an option to limit results to VA facilities and all locations
             | va        | 1     |
             | loc       | 0     |
             | rl        | 2     |
-				# Line below commented out temporarily due to change in utility file isWithinRadius. Should be enabled again once results page has api key properties change which should be in PR #430
+				# Line below commented out temporarily due to change in utility file isWithinRadius. Should be enabled again once results page has api key properties change which should be in Issue #403
         # And the 1 result item has a "Location:" info with "1390 locations, including 13 near you"
```

Comment:
You should be able to uncomment this and have it run successfully now

Reply:
uncommented